XIVSlothCombo/CustomComboPreset.cs

```diff
@@ -1033,6 +1033,19 @@ public enum CustomComboPreset
         [CustomComboInfo("oGCD Feature Everywhere", "Adds oGCD Feature to all other combos", RDM.JobID)]
         RedMageOgcdComboOnCombos = 13017,
 
+        [ParentCombo(SimpleRedMage)]
+        [CustomComboInfo("Simple Proc Fishing","Add Acceleration and Swiftcast to Simple Red Mage for proc fishing when necessary.", RDM.JobID)]
```

Comment:
Needs a better description I think.

Reply:
I agree, updated it with a better description.